shell/platform/darwin/macos/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -242,6 +247,27 @@ - (bool)testKeyEventsArePropagatedIfNotHandled {
   return true;
 }
 
+- (bool)testFlutterViewIsConfigured {
+  id engineMock = OCMClassMock([FlutterEngine class]);
+
+  id renderer_ = [[FlutterMetalRenderer alloc] initWithFlutterEngine:engineMock];
+  OCMStub([engineMock renderer]).andReturn(renderer_);
+
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:engineMock
+                                                                                nibName:@""
+                                                                                 bundle:nil];
+  [viewController loadView];
+
+  @try {
+    // Make sure "renderer" was called during "loadView", which means "flutterView" is created
+    OCMVerify([engineMock renderer]);
+  } @catch (...) {
+    return false;
+  }
```

Comment:
(note for @cbracken not @dubik) I know this pattern was copied from the other tests in this file but why are these tests returning `bool` (it's not even `BOOL`...) instead of failing in place?  For example this `@catch` hides the details of any exception instead of letting `XCTest` fail the test with the actual thing that happened.

Reply:
That's an excellent question that'll involve some archaeology. We should leave this for now and I'll follow-up with a cleanup PR once I figure out what the history of this was.